index.js

```diff
@@ -265,31 +265,40 @@ module.exports = {
     ],
     /* eslint-enable inclusive-language/use-inclusive-words -- Check for noninclusive words */
   },
-  overrides: [{
```

Comment:
diff is large due to odd original indentation (likely started as a single object, but indentation wasn't updated with the addition of another) 🤙